voxblox_ros/include/voxblox_ros/ros_params.h

```diff
@@ -0,0 +1,127 @@
+#ifndef VOXBLOX_ROS_ROS_PARAMS_H_
+#define VOXBLOX_ROS_ROS_PARAMS_H_
+
+#include <voxblox/core/esdf_map.h>
+#include <voxblox/core/tsdf_map.h>
+#include <voxblox/integrator/esdf_integrator.h>
+#include <voxblox/integrator/tsdf_integrator.h>
+
+namespace voxblox {
+
+inline TsdfMap::Config getTsdfMapConfigFromRosParam(
+    const ros::NodeHandle& nh_private) {
+  TsdfMap::Config tsdf_config;
+
+  // Workaround for OS X on mac mini not having specializations for float
+  // for some reason.
+  double voxel_size = tsdf_config.tsdf_voxel_size;
+  int voxels_per_side = tsdf_config.tsdf_voxels_per_side;
+  nh_private.param("tsdf_voxel_size", voxel_size, voxel_size);
+  nh_private.param("tsdf_voxels_per_side", voxels_per_side, voxels_per_side);
+  if (!isPowerOfTwo(voxels_per_side)) {
+    ROS_ERROR("voxels_per_side must be a power of 2, setting to default value");
+    voxels_per_side = tsdf_config.tsdf_voxels_per_side;
+  }
+
+  tsdf_config.tsdf_voxel_size = static_cast<FloatingPoint>(voxel_size);
+  tsdf_config.tsdf_voxels_per_side = voxels_per_side;
+
+  return tsdf_config;
+}
+
+inline TsdfIntegratorBase::Config getTsdfIntegratorConfigFromRosParam(
+    const ros::NodeHandle& nh_private) {
+  TsdfIntegratorBase::Config integrator_config;
+
+  integrator_config.voxel_carving_enabled = true;
+
+  // Used to be * 4 according to Marius's experience, now * 2.
```

Comment:
Can we update the comment, as it currently doesn't match the value.

Reply:
not sure what the valid message here is... (is it 4 because of my suggestion or because helen made it bigger again because she fixed the weighting? :P)

Reply:
Idk, maybe just drop the comment as people can look a few lines down and see its 4x

Reply:
agreed